src/mtcp/mtcp_restart.c

```diff
@@ -184,11 +184,11 @@ MTCP_PRINTF("Attach for debugging.");
 {int x=1; while(x);}
 #endif
 
-// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release.
+// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release, and after testing.
 // Without mtcp_check_vdso, CentOS 7 fails on dmtcp3, dmtcp5, others.
 #define ENABLE_VDSO_CHECK
 // TODO(karya0): Remove this block and the corresponding file after sufficient
-// testing.
+// testing:  including testing for __i386__, __arm__ and __aarch64__
 #ifdef ENABLE_VDSO_CHECK
```

Comment:
Is it possible for me to observe the 64-bit failure somewhere?

For the current timing, let's keep the code as is for 2.4.1, but open a new issue (targeted for 2.4.1) about replacing  `#ifdef ENABLE_VDSO_CHECK` with `#if defined(__i386__)`. I would like to figure out the problem and potentially fix it.